devservices/programs.conf

```diff
@@ -0,0 +1,3 @@
+[program:devserver]
+command=sentry devserver
```

Comment:
Is it possible to put quotes around the command? I find them useful when spaces are involved